src/Tcl/TclInterpreter.cpp

```diff
@@ -20,18 +20,364 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "TclInterpreter.h"
 
+#include <mutex>
+
 using namespace FOEDAG;
 
 #include <tcl.h>
 
+static std::string TclHistory() {
+  std::string script = R"(
+# history.tcl --
```

Comment:
Why not separate file? we will have a big mess with this

Reply:
There are a few Tcl files that will need to be sourced with the executable. It is best to embed those files in the executable itself. It will solve a lot of problems with Windows install for instance. Note that we can make those files standalone files. I'll create a separate file for this one.